### ISSUE TITLE
"running" is never set to "false"

### ISSUE THREAD

**Issue by jumpjack** [jnordberg/gif.js#128]
The only method setting "running" variable to "false" (after "render()" sets it to "true") is "abort()"; I think render() should set running to false once finished.

**Comment by mrrs878** [comment]
I did this: call gif.abort() in the callback function of gif.on("finished") and then running is reset to false

**Comment by Omiod** [comment]
To properly launch multiple time the gif rendering without issues, I had to do the following:
`gif.abort();
 gif.frames = [];
 delete gif.imageParts;
 delete gif.finishedFrames;
 delete gif.nextFrame;`

**Comment by mrrs878** [comment]
Yes, I found that there would be problems when rendering a new gif after calling `gif.abort()`, so I replaced `gif.abort()` with 
``` js
// ...
gif.running = false;
gif.frames = [];
// ...
```
